### ISSUE TITLE
Combine PicardMarkDuplicate and PicardRemoveDuplicate into single tool

### ISSUE THREAD

**Issue by syansanofi** [epam/fonda#197]
Is it possible to merge these two tools into a single one? For example:

`picard MarkDuplicates REMOVE_DUPLICATES=true`

can be used. 

Is there another situation where we need a separate samtools call to mark the duplicate?

Additionally CREATE_INDEX=true can also be used to replace the indexing of duplicate reads

`picard MarkDuplicates REMOVE_DUPLICATES=true --CREATE_INDEX=true`

**Comment by kamyshova** [comment]
@syansanofi Yes, it's possible.  We apply PicardRemoveDuplicate if `rmdup` flag is on or these are `scRnaExpression_Fastq` or `capture` or `WGS` workflows. Should we preserve this logic?
